libraries/package-deps-hash/src/getRepoState.ts

```diff
@@ -246,6 +246,109 @@ export function applyWorkingTreeState(
   }
 }
 
+/**
+ * Get the list of all git submodules path
+ * @param rootDirectory - The root directory of the Git repository
+ * @param gitPath - The path to the Git executable
+ * @internal
```

Comment:
Are these exported? Do the internals make sense?

Reply:
If these are internal and exported, the function names should start with an underscore

Reply:
I make theses two new functions as unexported.